params/config.go

```diff
@@ -344,7 +347,7 @@ type NodeConfig struct {
 }
 
 // NewNodeConfig creates new node configuration object
-func NewNodeConfig(dataDir string, clstrCfgFile string, networkID uint64) (*NodeConfig, error) {
+func NewNodeConfig(dataDir, clstrCfgFile, fleet string, networkID uint64) (*NodeConfig, error) {
```

Comment:
With 4 parameters, semantics of `NewNodeConfig()` becomes less readable. Maybe worth rethinking how we use this node config stuff (in separate PR)? I.e. `params.NewNodeConfig("", "", "", 0)` is not the extra clear :)

Reply:
I agree. I think the whole `params` package needs some rethinking.